WooCommerce/Classes/ViewRelated/Blaze/ConfirmPayment/BlazePaymentMethodsView.swift

```diff
@@ -53,6 +59,20 @@ struct BlazePaymentMethodsView: View {
                 BlazeAddPaymentMethodWebView(viewModel: viewModel)
             }
         })
+        .alert(Localization.LoadPaymentMethodsErrorAlert.paymentMethods, isPresented: $viewModel.showLoadPaymentsErrorAlert) {
+            Button(Localization.LoadPaymentMethodsErrorAlert.cancel, role: .cancel) { }
+
+            Button(Localization.LoadPaymentMethodsErrorAlert.retry) {
+                Task {
+                    await viewModel.reloadPaymentMethods()
+                }
+            }
+        }
+        .onAppear {
+            Task {
+                await viewModel.reloadPaymentMethods()
+            }
+        }
```

Comment:
Nit: this can be replaced with `.task` modifier:

```suggestion
        .task {
            await viewModel.reloadPaymentMethods()
        }
```

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/12321/commits/7e235bf9c7dfc5994ce4951557ce819b50e00fb8